INSTALL.md

```diff
@@ -1036,17 +1038,17 @@ for the following:
 
     AR, CC, CXX, CROSS_COMPILE, HASHBANGPERL, PERL, RANLIB, RC, and WINDRES
 
-For example, the following command will not see -DBAR:
+For example, the following command will not see `-DBAR`:
 
     CPPFLAGS=-DBAR ./config -DCOOKIE
 
 However, the following will see both set variables:
 
     CC=gcc CROSS_COMPILE=x86_64-w64-mingw32- ./config -DCOOKIE
 
-If CC is set, it is advisable to also set CXX to ensure both the C and C++
+If `CC `is set, it is advisable to also set `CXX` to ensure both the C and C++
```

Comment:
Space should be outside the backticks around CC

Reply:
Fixed